src/main/java/com/stripe/StripeClient.java

```diff
@@ -570,4 +573,59 @@ StripeResponseGetterOptions buildOptions() {
           connectBase);
     }
   }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @return the JSON response as a string
+   */
+  @Deprecated
+  public StripeResponse rawRequest(
+      final ApiResource.RequestMethod method, final String relativeUrl, final String content)
+      throws StripeException {
+    RawRequestOptions options = RawRequestOptions.builder().build();
+    return rawRequest(method, relativeUrl, content, options);
+  }
+
+  /**
+   * Send raw request to Stripe API. This is the lowest level method for interacting with the Stripe
+   * API. This method is useful for interacting with endpoints that are not covered yet in
+   * stripe-java.
+   *
+   * @param method the HTTP method
+   * @param relativeUrl the relative URL of the request, e.g. "/v1/charges"
+   * @param content the body of the request as a string
+   * @param options the special modifiers of the request
+   * @return the JSON response as a string
+   */
+  public StripeResponse rawRequest(
+      final ApiResource.RequestMethod method,
+      final String relativeUrl,
+      final String content,
+      final RawRequestOptions options)
```

Comment:
check options for null? or default

Reply:
Done in [Deserialize takes in responseGetter](https://github.com/stripe/stripe-java/pull/1728/commits/7ae40fb7f986ec72627da053615634d311a86057)

Reply:
Was it? The `options.getApiMode` call will still nullref if options is null.

Reply:
Good catch - must have screwed up my git. Here's the change plus a test: [Handle null RawRequestOptions better](https://github.com/stripe/stripe-java/pull/1728/commits/1332ee7460b7cf7652fa57735b32550ddfe85230)